Frontend/ui-customer/src/Resources/Localizable/Language/KN.purs

```diff
@@ -222,6 +222,7 @@ getKN script = case script of
   CHANGE_DROP_LOCATION -> "ಡ್ರಾಪ್ ಸ್ಥಳವನ್ನು ಬದಲಾಯಿಸಿ"
   YOU_CAN_TAKE_A_WALK_OR_CONTINUE_WITH_RIDE_BOOKING -> "ನೀವು ನಡೆಯಬಹುದು ಅಥವಾ ರೈಡ್ ಬುಕ್ ಮಾಡುವುದನ್ನು ಮುಂದುವರಿಸಬಹುದು"
   YOUR_TRIP_IS_TOO_SHORT_YOU_ARE_JUST -> "ನಿಮ್ಮ ಪ್ರವಾಸವು ತುಂಬಾ ಚಿಕ್ಕದಾಗಿದೆ. ನಿಮ್ಮ ಗಮ್ಯಸ್ಥಾನದಿಂದ ನೀವು ಕೇವಲ "
+  THE_DESTINATION_IS_TOO_CLOSE -> "ಆಟೋವನ್ನು ಬುಕ್ ಮಾಡಲು ತಲುಪಬೇಕಾದ ಸ್ಥಳವು ತುಂಬಾ ಹತ್ತಿರದಲ್ಲಿದೆ. ನೀವು ಬೇರೆ ಪಿಕಪ್ ಮತ್ತು ಡ್ರಾಪ್ ಹೊಂದಿರುವಿರಾ ಎಂಬುದನ್ನು ಖಚಿತಪಡಿಸಿಕೊಳ್ಳಿ."
```

Comment:
Update this
"ಗಮ್ಯಸ್ಥಾನವು ತುಂಬಾ ಹತ್ತಿರದಲ್ಲಿದೆ. ನೀವು ಬೇರೆ ಪಿಕಪ್ ಮತ್ತು ಡ್ರಾಪ್ ಹೊಂದಿರುವಿರಾ ಎಂಬುದನ್ನು ಖಚಿತಪಡಿಸಿಕೊಳ್ಳಿ."